build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -408,6 +419,14 @@ class SeleniumWebDriverController {
       this.getWaitFn_(() => webElement.isEnabled())
     );
   }
+  /**
+   * @return {!Promise<!ElementHandle>}
+   * @override
+   */
+  async getAllWindows() {
+    const handles = await this.driver.getAllWindowHandles();
+    return handles.map(handle => new ElementHandle(handle, this));
```

Comment:
No need to turn this into an array of `ElementHandles`. Let's keep this to a string array.